atomic_reactor/constants.py

```diff
@@ -160,6 +160,9 @@
 MEDIA_TYPE_OCI_V1 = "application/vnd.oci.image.manifest.v1+json"
 MEDIA_TYPE_OCI_V1_INDEX = "application/vnd.oci.image.index.v1+json"
 
+# PNC related constants
+PNC_SYSTEM_USER = "newcastle"
```

Comment:
Can you elaborate more about this constant? Is it really a constant?

Reply:
yes, this is a constant we need to identify if the brew build was created by PNC. Based on that we follow different logic to look for its sources.

Reply:
and this is hardcoded to every PNC and cannot be changed?

Reply:
yes, this is the service account and all production builds are done using this account.